### ISSUE TITLE
Error because jquery function $().remove was not complete in welcome.txt parsing

### ISSUE THREAD

**Issue by MichaelDimmitt** [motsu35/seeingbinary-web-code#1]
Hello Motsu, 
Really enjoy this website, I know it has been a long time since you have made a change to the project, did you reverse engineer a larger project when you built it?
Looking for other websites that do this well (terminal emulator).

Im in the process of reverse engineering your website to one of mine. I will link you credit in a description somewhere on the website.
This is the best site I have seen that serves as as a terminal emulator.

1) glitch.js when logged in as root == 👌 
2) addText function == 💯 

## The Issue: 
https://seeingbinary.com/ has 3 errors in the console.log

They are caused in the addText function, specifically the jquery of welcome.txt:
```javascript
let Typer.index = 0
let Typer.speed = 2
Typer.index+=Typer.speed
var text=Typer.text.substring(0,Typer.index)
$("#console").html(text.replace(rtn,"<br/>"));
```
```
As shown below the browser is calling $( ... but does not complete the script.
```
![errorMessage](https://user-images.githubusercontent.com/11463275/54628381-cc2cc700-4a4b-11e9-9d03-804a9c08d50d.png)

## Fix no.1: 
Wrap $("#console").html(text) in a try catch. Pull Request: #2 

## Fix no.2:
Welcome .txt right now is stored in a single variable.
The file could be separated into a first section leading up to the script section, 
The script section that removes the previous section and then the final section.

Inside the t function: 
```javascript
Typer.section = 'section1'
Typer.text='file1'
var timer = setInterval("t();", 30);
function t() {

  if(Typer.section == 'section1' || Typer.section == 'section2' )
  {
    Typer.addText({"keyCode": 123748});

    if(Typer.section == 'section1' && Typer.index > Typer.text.length) {
      $("#console").html('<script>$('span[id="fake_login"]').remove();</script>');
      Typer.text = 'file2'
      Typer.section = 'section2';
      Typer.index=0;
    }

    if (Typer.index > Typer.section == 'section2' && Typer.text.length) {
	clearInterval(timer);
    }
  }
```

**Comment by motsu35** [comment]
Yeah, the part with the scrolling text was another project, This was many years ago, so i dont quite remember, but im fairly sure the scrolling text came from a project someone was showing off on youtube. I adapted the scrolling part to clear, and then implemented the typing bit afterwards, which is also why the code kind of is in two parts, the scrolling welcome page, and then the interactive part.

If i was to do it again, i would make a class per command, and implement it similar to how the 'cat' command works. It checks to see if the file you want to open is an actual file or a function, and then calls it if it where a function. I would do that, but with classes, where it passes the arguments in so each command can be split off into its own file. might be worth doing if you are making a large amount of commands.

as far as the project goes, yeah - it was kind of a project i used to learn javascript, then didnt work on it any more after that. I appreciate the PR!

**Comment by MichaelDimmitt** [comment]
@motsu35, I found another site that has a cool emulator.
https://repl.it/site/jobs

repl.it's job board site.
It makes sense their emulator would support a bunch of commands.
